src/MainWindow.cpp

```diff
@@ -840,7 +840,10 @@ void MainWindow::doubleClickTable(const QModelIndex& index)
     editDock->allowEditing(allowEditing);
 
     // Load the current value into the edit dock
-    editDock->loadText(index.data(Qt::EditRole).toByteArray(), index.row(), index.column());
+    QByteArray cellData = index.data(Qt::EditRole).toByteArray();
+    int newRow = index.row();
```

Comment:
Isn't this `cellRow`? So the row index of the selected table cell in the `Browse Data` tab? Just asking